src/Maker/MakeUser.php

```diff
@@ -152,19 +156,25 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             $classPath = $this->entityClassGenerator->generateEntityClass(
                 $userClassNameDetails,
                 false, // api resource
-                $userClassConfiguration->hasPassword() && interface_exists(PasswordUpgraderInterface::class) // security user
+                $userClassConfiguration->hasPassword() && interface_exists(PasswordUpgraderInterface::class) // security user,
```

Comment:
revert, maker-bundle is compatible with php 7.1,  
trailing comma are allowed in +7.3

+ do not change things that are not related to your PR

Reply:
done